### PR TITLE
remove obsolete files

### PR DESCRIPTION
**Pull request by jcantrill**
This PR removes obsolete files from fluentd as they are provided by CLO:
* configuration
* run.sh

cc @alanconway @ewolinetz

### CODE REVIEW

**Comment by vimalk78** [comment]
We need to do the same Dockerfile change in midstreams fluentd Dockerfile
https://code.engineering.redhat.com/gerrit/gitweb?p=openshift-logging.git;a=blob;f=distgit/containers/logging-fluentd/Dockerfile.in;h=b7017be573f7d4ee9ae0cd1c5a5178de50e7e1a3;hb=refs/heads/private-rhaos-4.6-rhel-7

**Comment by vimalk78** [comment]
> We need to do the same Dockerfile change in midstreams fluentd Dockerfile
> https://code.engineering.redhat.com/gerrit/gitweb?p=openshift-logging.git;a=blob;f=distgit/containers/logging-fluentd/Dockerfile.in;h=b7017be573f7d4ee9ae0cd1c5a5178de50e7e1a3;hb=refs/heads/private-rhaos-4.6-rhel-7

I will make this change

File: fluentd/run.sh [hunk 1]
```diff
@@ -1,280 +0,0 @@
-#!/bin/bash
```
Comment (ewolinetz) [inline fluentd/run.sh:1]: given we still have this as the Dockerfile `CMD` it is probably a good practice to keep this file, but we can update it to be more of a `usage` file in that it simply echos out the usage for this (saying it should be used in concert with CLO and a LogForwarding CR).

**Comment by ewolinetz** [comment]
one comment

File: fluentd/run.sh (hunk 1 above)
Comment (jcantrill) [reply fluentd/run.sh:1]: restored run.sh, truncated the content

**Comment by ewolinetz** [comment]
/hold
tests are spinning and at first glance it could be related to changes here
```
WARN: 0 of 6 fluentd Running
WARN: 0 of 6 fluentd Running
WARN: 0 of 6 fluentd Running
WARN: 0 of 6 fluentd Running
WARN: 0 of 6 fluentd Running
[ERROR] PID 1757: test/multi_tenancy.sh:72: `${repo_dir}/hack/deploy-logging.sh` exited with status 1.
```

**Comment by jcantrill** [comment]
closing in favor of #2108

**Pull request closed without merging**